### ISSUE TITLE
Add support for IDrive

### ISSUE THREAD

**Issue by atqhg23** [cryptomator/ios#275]
### Please agree to the following

- [X] I have searched [existing issues](https://github.com/cryptomator/ios/issues?q=) for duplicates
- [X] I agree to follow this project's [Code of Conduct](https://github.com/cryptomator/ios/blob/develop/.github/CODE_OF_CONDUCT.md)

### Summary

Add support for IDrive on Cryptomator 2

### Motivation

Hi,

Please add support for IDrive on Cryptomator 2. From what I found, IDrive does not support WebDAV ([link to source](https://community.cryptomator.org/t/webdav-urls-of-common-cloud-storage-services/75/22?page=2)). This would help out a lot. Thank you!

### Considered Alternatives

I tried through WebDAV but IDrive does not support it.

**Comment by tobihagemann** [comment]
I don't think that this is possible. I couldn't find any developer documentation on how to integrate an official or public API for IDrive. They seem to have a different product called IDrive e2 though, which would already work with Cryptomator via S3.

You may try to contact IDrive and let them know that this is something that you would like to have from them.
